packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -124,6 +135,49 @@ module.exports = ({ types: t }) => {
         ],
       },
 
+      AssignmentExpression(path) {
+
+        const right = path.get('right');
+        const left = path.get('left');
+
+        const canShorten = (
+          right.type === 'BinaryExpression' &&
+          operators.has(right.node.operator) &&
+          left.node.name === right.node.left.name
```

Comment:
requires `isIdentifier` check. if `left.node` and `right.node.left` are not identifiers, it can end up in `undefined === undefined` check and canShorten will be true - eg: `[a,b] = foo() + 1` - canShorten will be true for this.